components/tidb_query_common/Cargo.toml

```diff
@@ -5,23 +5,24 @@ edition = "2018"
 publish = false
 description = "Common utility of a query engine to run TiDB pushed down executors"
 
+[features]
+protobuf-codec = ["kvproto/protobuf-codec"]
+prost-codec = ["kvproto/prost-codec"]
+
 [dependencies]
-hex = "0.3"
-rand = "0.6.5"
-time = "0.1"
+derive_more = "0.99.3"
```

Comment:
Better not to sort the dependency. On the one hand, it's hard to review the changes, on the other hand, it makes cherry-pick with dependency change hard to succeed.